static/app/views/explore/logs/tables/logsInfiniteTable.tsx

```diff
@@ -96,17 +96,13 @@ export function LogsInfiniteTable({
   const [isFunctionScrolling, setIsFunctionScrolling] = useState(false);
 
   const sharedHoverTimeoutRef = useRef<NodeJS.Timeout | null>(null);
-  const {initialTableStyles, onResizeMouseDown} = useTableStyles(
-    fields.length,
-    tableRef,
-    {
-      minimumColumnWidth: 50,
-      prefixColumnWidth: 'min-content',
-      staticColumnWidths: {
-        [OurLogKnownFieldKey.MESSAGE]: '1fr',
-      },
-    }
-  );
+  const {initialTableStyles, onResizeMouseDown} = useTableStyles(fields, tableRef, {
+    minimumColumnWidth: 50,
+    prefixColumnWidth: 'min-content',
+    staticColumnWidths: {
+      [OurLogKnownFieldKey.MESSAGE]: '1fr',
+    },
```

Comment:
I didn't check the context for the PR to change to counts vs. the actual fields but if you want to keep count you can use index with `[fields.indexOf(OurLogKnownFieldKey.MESSAGE)]: '1fr'` here